cardshifter-core/src/main/java/com/cardshifter/core/groovy/EffectDelegate.groovy

```diff
@@ -230,7 +234,14 @@ class EffectDelegate {
             String targetStr = '';
             Closure closure = null;
             if (who == targets) {
-                targetStr = 'targets'
+                // TODO: How to reference the current entity?
```

Comment:
@Zomis Do you know? Below I've tried using `entity()`, but that doesn't work here. It seems like the source entity is only available at perform, but not on "setup" where descriptions are generated. The whole problem stems from the description requiring information from both the effect and filter delegates. Maybe a completely different solution is better?